.github/workflows/ci.yml

```
@@ -60,15 +57,20 @@ jobs:
           path: coverage.xml
 
       - name: Install CodeScene Coverage CLI
-        if: env.CS_ACCESS_TOKEN != ''
+        if: ${{ secrets.CS_ACCESS_TOKEN != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)